.setup/update_system.sh

```diff
@@ -27,6 +27,10 @@ fi
 #install DLL for zbar
 apt-get install libzbar0 --yes
 
+#libraries for comment counting :
+#install cloc
+apt install cloc --yes
```

Comment:
Per the [Debian Administrator's Handbook on the apt tools](https://debian-handbook.info/browse/stable/sect.apt-get.html):
> Both tools are built on top of the same library and are thus very close, but the default behavior of **apt** has been improved for interactive use and to actually do what most users expect. The APT developers reserve the right to change the public interface of this tool to further improve it. On the opposite, the public interface of **apt-get** is well defined and will not change in any backwards incompatible way. It is thus the tool that you want to use when you need to script package installation requests.

so we should do instead:
```suggestion
apt-get install cloc --yes
```